### PR TITLE
[OWL-1215] <Button /> component

### PR DESCRIPTION
**Pull request by CApopsicle**
Button Component - without button group
http://jira.cepave.com/browse/OWL-1215
http://design.cepave.com/owl-ui-lite/docs/buttons/index.html

- [x] Button component
- [x] Test Specs
- [x] Doc

### CODE REVIEW

File: src/client/views/button/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  render(h) {
+    return (
+      <div>
+        <Markdown src={require('./doc.md')} />
+        <div class={[s.buttonWrapper]}>
+          <Button icon='plus' content="新增公告" />
+          <Button status="primary" content="送出" />
+          <Button status="outline" content="搜尋" />
+          <Button status="primaryOutline" content="关闭"/>
```
Comment (chuanxd) [inline src/client/views/button/index.js]: prefer zh-TW and space

File: src/client/views/button/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        </div>
+        <span>Disabled buttons</span>
+        <div class={[s.buttonWrapper]}>
+          <Button content="Submit" disabled={true}/>
```
Comment (chuanxd) [inline src/client/views/button/index.js]: checking your lint

**Review by chuanxd — CHANGES_REQUESTED** [review]

File: src/client/views/button/index.js (hunk 1 above)
Comment (rwu823) [reply src/client/views/button/index.js]:
``` jsx
// better 接口
<Button>
 新增公告
</Button>
```

File: src/components/button/index.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  render(h) {
+    const { status, content, icon, style, fill } = this
+
+    return (
+      <button type="button" class={style}>
+        {icon
+          ? <Icon class={[s.icon]} typ={icon} size={18}/>
+          : ''
+        }
+        <span>{content}</span>
```
Comment (rwu823) [inline src/components/button/index.js]: 這邊應該給外面用的人自己放 Icon 就好，比較靈活

**Review by rwu823 — CHANGES_REQUESTED** [review]

File: src/components/button/index.js (hunk 3 above)
Comment (chuanxd) [reply src/components/button/index.js]: QQ

File: src/client/views/button/index.js (hunk 1 above)
Comment (CApopsicle) [reply src/client/views/button/index.js]: :ok_hand::ok_hand:

File: src/components/button/index.js (hunk 3 above)
Comment (CApopsicle) [reply src/components/button/index.js]: :ok_hand::ok_hand: 已修改

**Review by rwu823 — APPROVED** [review]

File: src/components/button/index.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import Icon from '../icon'
```
Comment (chuanxd) [inline src/components/button/index.js]: 沒用到移掉

**Review by chuanxd — CHANGES_REQUESTED** [review]

File: src/components/button/index.js (hunk 4 above)
Comment (CApopsicle) [reply src/components/button/index.js]: :ok_hand::ok_hand:

**Review by chuanxd — APPROVED** [review]

**Pull request merged**